packages/engine/Source/Scene/PostProcessStage.js

```diff
@@ -93,9 +93,11 @@ import PostProcessStageSampleMode from "./PostProcessStageSampleMode.js";
  */
 function PostProcessStage(options) {
   options = defaultValue(options, defaultValue.EMPTY_OBJECT);
-  const fragmentShader = options.fragmentShader;
-  const textureScale = defaultValue(options.textureScale, 1.0);
-  const pixelFormat = defaultValue(options.pixelFormat, PixelFormat.RGBA);
+  const {
```

Comment:
Good point. We do have to consider `null` when working with DOM elements, since [getElementById](https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById) etc. may return `null` if nothing is found.

In this case, I think the destructuring with defaults is OK, because:
- The user would have to explicitly set `null` as a value. (Leaving it blank would result in the default being set).
- We immediately check the type of these options, so the user would get a meaningful error message if they set a value to 'null'.